src/components/ArtifactCollections.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="artifact-collections">
+    <div class="artifact-collections__filters">
+      <ResultsCount :label="localization.info.artifact" :count="artifactsCount" class="artifact-collection-list__results" />
+      <SearchInput v-model="searchTerm" :placeholder="localization.info.artifactSearch" :label="localization.info.artifactSearch" class="artifact-collection-list__search" />
+      <ArtifactTypeSelect v-model:selected="selectedType" class="artifact-collection-list__type" />
+      <ViewModeButtonGroup class="artifact-collections__view-mode-button-group" />
+    </div>
+
+    <RowGridLayoutList :items="artifacts">
+      <template #default="{ item }: { item: Artifact }">
+        <ArtifactCard :artifact="item" />
+      </template>
+
+      <template #empty>
+        <ArtifactCollectionsEmptyState />
+      </template>
+    </RowGridLayoutList>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed, ref } from 'vue'
+  import { ArtifactTypeSelect, ResultsCount, SearchInput } from '@/components'
+  import ArtifactCard from '@/components/ArtifactCard.vue'
+  import ArtifactCollectionsEmptyState from '@/components/ArtifactCollectionsEmptyState.vue'
+  import RowGridLayoutList from '@/components/RowGridLayoutList.vue'
+  import ViewModeButtonGroup from '@/components/ViewModeButtonGroup.vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { localization } from '@/localization'
+  import { ArtifactsFilter, ArtifactType, Artifact } from '@/models'
```

Comment:
```suggestion
  import { ArtifactsFilter, ArtifactType } from '@/models'
```

Reply:
This is used in the template, eslint is being a bum :(